### ISSUE TITLE
Many elements inside data table are not accessible using keyboard

### ISSUE THREAD

**Issue by hannyle** [CSCfi/swift-browser-ui#889]
#### Describe the bug

Many elements inside data table, for example: nested dropdown menu, collapsed menu, etc. are not accessible by using keyboard.

#### To Reproduce
Steps to reproduce the behaviour:
1. Go to SD Connect's browser page and make the screen smaller so some columns are hidden
2. Use tab to navigate to data table
4. Try to reach the hidden content using keyboard
5. Cannot access the hidden items

#### Expected behavior

All elements inside data table are accessible in normal screen sizes and smaller/mobile screen sizes
